riscv/insns/flh.h

```diff
@@ -1,3 +1,3 @@
-require_extension(EXT_ZFHMIN);
+require_either_extensions(EXT_ZFHMIN, EXT_ZFBFMIN, EXT_ZVFBFMIN);
```

Comment:
An alternative approach here would be to define a new Spike-internal extension, say, `EXT_INTERNAL_ZFH_MOVE`, which includes only these four instructions.  Then we would enable this extension when any of those three extensions is included.  (We would not add any command-line support.)

I could go either way on this.  What do you think, @scottj97?

Reply:
The alternate approach would also avoid the copy-pasta in the disassembler: https://github.com/riscv-software-src/riscv-isa-sim/pull/1321/commits/107ee46f29f0a531a7542bc48537cfd0d565c067#diff-6ec72161b4427824dd85867de3c0998cd73d7ebfaad6d7af568240a38767b58dR1243

Reply:
It's acceptable to me.